libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -32,16 +32,16 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
 
     // @Param: REQUIRE
     // @DisplayName: Require Arming Motors 
-    // @Description: Arming disabled until some requirements are met. If 0, there are no requirements (arm immediately).  If 1, require rudder stick or GCS arming before arming motors and sends the minimum throttle PWM value to the throttle channel when disarmed.  If 2, require rudder stick or GCS arming and send 0 PWM to throttle channel when disarmed. See the ARMING_CHECK_* parameters to see what checks are done before arming. Note, if setting this parameter to 0 a reboot is required to arm the plane.  Also note, even with this parameter at 0, if ARMING_CHECK parameter is not also zero the plane may fail to arm throttle at boot due to a pre-arm check failure.
-    // @Values: 0:Disabled,1:THR_MIN PWM when disarmed,2:0 PWM when disarmed
+    // @Description: Arming disabled until some requirements are met. If 0, there are no requirements (arm immediately after ARMING_CHECKS passed).  If 1 or 2, require rudder stick or GCS arming before arming motors.
+    // @Values: 0:Arm Immediately,1:THR_MIN (or SERVO_TRIM for reverse thrust) when disarmed,2:0 PWM when disarmed
```

Comment:
If this requires a reboot then it should be marked as such.

Reply:
I don't know if it does or doesn't; I was just clarifying some of the language. I don't think it requires a re-boot, however will you please make changes as appropriate so I (or you) can commit them?